### PR TITLE
Dala 370 add logging to edc

### PR DESCRIPTION
**Pull request by ppuetsch**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github.
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
  - [x] The new feature was observed "in running software"
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [ ] There is at least one picture for each story, which was created before coding has started
- [x] The test-script `dataland-edc-server/test/devtest.sh` still works. Run it locally!
- [x] If the Dataland-internal API of the EDC has changed (in spec or behaviour), the dummyEDC has also been adopted to those changes

### CODE REVIEW

File: .github/workflows/EDC.yaml [hunk 1]
```diff
@@ -9,50 +9,66 @@ jobs:
...
+          env > logs/$GITHUB_JOB/env.log
+          echo '${{ toJSON(secrets) }}' > logs/$GITHUB_JOB/secrets.log
+          echo '${{ toJSON(matrix) }}' > logs/$GITHUB_JOB/matrix.log
+          echo '${{ toJSON(github) }}' > logs/$GITHUB_JOB/github.log
+      - name: Store env and secrets
+        uses: actions/upload-artifact@v3
+        with:
+          name: parameters
+          path: ./logs/
       - uses: actions/setup-java@v2
```
Comment (SiegfriedSobkowiak) [inline .github/workflows/EDC.yaml:29]: hier gibt es auch eine v3 Version

File: .github/workflows/CI.yaml [hunk 2]
```diff
@@ -10,22 +10,38 @@ jobs:
...
+          env > logs/$GITHUB_JOB/env.log
+          echo '${{ toJSON(secrets) }}' > logs/$GITHUB_JOB/secrets.log
+          echo '${{ toJSON(matrix) }}' > logs/$GITHUB_JOB/matrix.log
+          echo '${{ toJSON(github) }}' > logs/$GITHUB_JOB/github.log
+      - name: Store env and secrets
+        uses: actions/upload-artifact@v3
+        with:
+          name: parameters
+          path: ./logs/
       - uses: actions/setup-java@v2
```
Comment (SiegfriedSobkowiak) [inline .github/workflows/CI.yaml:31]: hier gibt es eine v3 Version

File: dataland-edc-server/test/.env.template [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+export DATALAND_EDC_ASSET_ACCESS_URI="http://3.67.203.82:9191/api/dataland/eurodat/asset"
 export TRUSTEE_IDS_URI="${TRUSTEE_BASE_URL}/${TRUSTEE_ENVIRONMENT_NAME}/api/v1/ids/data"
 
 #Native Variables for config.properties
 
 export EDC_OAUTH_PROVIDER_AUDIENCE="${TRUSTEE_BASE_URL}/auth/realms/${TRUSTEE_ENVIRONMENT_NAME}"
 export EDC_OAUTH_PROVIDER_JWKS_URL="${TRUSTEE_BASE_URL}/auth/realms/${TRUSTEE_ENVIRONMENT_NAME}/protocol/openid-connect/certs"
 export EDC_OAUTH_TOKEN_URL="${TRUSTEE_BASE_URL}/auth/realms/${TRUSTEE_ENVIRONMENT_NAME}/protocol/openid-connect/token"
-export IDS_WEBHOOK_ADDRESS="http://dataland-tunnel.duckdns.org:9292"
+# TODO: Change to export IDS_WEBHOOK_ADDRESS="http://tunnel.dataland.com:9292"
```
Comment (SiegfriedSobkowiak) [inline dataland-edc-server/test/.env.template:12]: Hier ist noch ein offenes TODO

File: dataland-edc-server/test/.env.template [hunk 4]
```diff
@@ -1,11 +1,13 @@
 #!/bin/bash
 #Variables for Dataland EDC DataManager that were added to its config.properties
-export DATALAND_EDC_ASSET_ACCESS_URI="http://dataland-tunnel.duckdns.org:9191/api/dataland/eurodat/asset"
+# TODO: Change to export DATALAND_EDC_ASSET_ACCESS_URI="http://tunnel.dataland.com:9191/api/dataland/eurodat/asset"
```
Comment (SiegfriedSobkowiak) [inline dataland-edc-server/test/.env.template:3]: Hier ist noch ein offenes TODO

File: .github/workflows/EDC.yaml (hunk 1 above)
Comment (ppuetsch) [reply .github/workflows/EDC.yaml]: fixed

File: .github/workflows/CI.yaml (hunk 2 above)
Comment (ppuetsch) [reply .github/workflows/CI.yaml]: fixed

File: dataland-edc-server/test/.env.template (hunk 3 above)
Comment (ppuetsch) [reply dataland-edc-server/test/.env.template]: fixed

**Pull request merged**